src/client/java/dev/creesch/config/ModConfigScreen.java

```diff
@@ -1,39 +1,93 @@
+                                        .controller(BooleanControllerBuilder::create)
+                                        .build()
+                                )
+                                .build()
+                        )
+                        .group(
+                            ListOption.<String>createBuilder()
+                                .name(Text.literal("Extra Ping Keywords"))
+                                .description(
+                                    OptionDescription.of(Text.literal(
+                                        "Extra keywords to ping on.\n" +
+                                        "This will ping the browser window any time one of these words appear in the chat."
+                                    ))
+                                )
+                                .binding(
+                                    ModConfig.HANDLER.defaults().pingKeywords,
+                                    () -> ModConfig.HANDLER.instance().pingKeywords,
+                                    val -> ModConfig.HANDLER.instance().pingKeywords = val
+                                )
+                                .controller(StringControllerBuilder::create)
+                                .initial("")
+                                .build()
+                        )
+                        .build()
+                )
+                .category(
+                    ConfigCategory.createBuilder()
                         .name(Text.literal("Network Settings"))
-                        .group(OptionGroup.createBuilder()
+                        .group(
+                            OptionGroup.createBuilder()
                                 .name(Text.literal("Port Settings"))
-                                .option(Option.<Integer>createBuilder()
+                                .option(
+                                    Option.<Integer>createBuilder()
                                         .name(Text.literal("HTTP Port"))
-                                        .description(OptionDescription.of(Text.literal("""
-                                                Port number used to serve the web interface.
-                                                Make sure that both this port and the one above it (+1) are available.
-                                                
-                                                IMPORTANT: You need to restart minecraft for this to take effect.""")))
+                                        .description(
+                                            OptionDescription.of(Text.literal(
+                                                "Port number used to serve the web interface.\n" +
+                                                "Make sure that both this port and the one above it (+1) are available.\n" +
+                                                "IMPORTANT: You need to restart minecraft for this to take effect."

```

Comment:
any reason you are changing this? Don't really mind, but there need to be two line breaks before this sentence ;)

Reply:
I was finding it hard to find matching parentheses.